drivers/hmis/lib/form_data/allegheny/occurrence_point_forms/unit_assignment.json

```diff
@@ -0,0 +1,24 @@
+{
+  "item": [
+    {
+      "type": "CHOICE",
+      "link_id": "unit-assignment",
+      "text": "Unit Assignment",
+      "mapping": {
+        "record_type": "ENROLLMENT",
+        "field_name": "currentUnit"
+      },
+      "pick_list_reference": "AVAILABLE_UNITS_FOR_ENROLLMENT",
+      "_comment": "Hacky way to disallow changing unit after exit. It would be better if the most recent unit was displayed, or show full unit assignment history. Exited enrollments shouldnt have currently assigned units.",
```

Comment:
Couldn't you show the current unit as read-only (or maybe this as read-only) when the "enable when" conditions are met?

Reply:
This does show the current unit as read-only when conditions are met. I guess it's not "hacky" its just not the ideal display. For Exited enrollments, this will always display as `Unit Assignment:    Data Not Collected` because units are only assigned during the duration of the enrollment.